apps/server/src/app/api/getVideo/route.ts

```diff
@@ -0,0 +1,24 @@
+import { NextResponse } from 'next/server';
+
+import { uriToBlob } from '~/utils/blob';
+
+export async function POST(req: Request) {
+  try {
+    const formData = await req.formData();
+
+    const uri = formData.get('uri') as string;
+    if (uri) {
+      // const blob = await uriToBlob(body.url);
+      const res = await fetch(uri);
+      const blob = await res.blob();
+      const headers = new Headers();
+      headers.set('Content-Type', 'video/*');
+      return new NextResponse(blob, { status: 200, statusText: 'OK', headers });
+    }
+
+    return NextResponse.json({ status: 'fail', error: 'No URL' });
+  } catch (e) {
+    console.error(e);
+    return NextResponse.json({ status: 'fail', error: e });
+  }
```

Comment:
rename to "uploadVideoToS3"

Reply:
renamed to "uriToBlob" (this exactly what this route api do)